### PR TITLE
Reger men/reg v1

### PR DESCRIPTION
**Pull request by reger-men**
Support save `tensor_to_image` as image file.
Fix `U008` output size problem.
Fix some small bugs for reading `U008` images.

### CODE REVIEW

**Comment by rgiduthuri** [comment]
See too many changes with the diff because of changes to whitespaces.
Could you please recommit with just the lines that need change?
This will help me review the changes easily.

**Comment by reger-men** [comment]
done.

File: runvx/vxUtils.cpp [hunk 1]
```diff
@@ -571,6 +571,43 @@ int WriteImage(vx_image image, vx_rectangle_t * rectFull, FILE * fp)
...
+            printf("ERROR: display of image type (%4.4s) is not support. Exiting.\n", (const char *)&im_format);
+            throw - 1;
+        }
+
+        auto img = cv::Mat(im_height, im_width, im_type, src, addr.stride_y);
+        imwrite(fileName, img);
+
+        ERROR_CHECK(vxCommitImagePatch(image, rectFull, plane, &addr, src));
+    }
+}
```
Comment (rgiduthuri) [inline runvx/vxUtils.cpp]: This function needs to return a value in all code paths. Need to add "return 0;" at the end.

File: runvx/vxUtils.cpp [hunk 2]
```diff
@@ -571,6 +571,43 @@ int WriteImage(vx_image image, vx_rectangle_t * rectFull, FILE * fp)
 	return 0;
 }
 
+// write image compressed
+int WriteImageCompressed(vx_image image, vx_rectangle_t * rectFull, const char * fileName) {
+    // get number of planes, image width in bytes for single plane
```
Comment (rgiduthuri) [inline runvx/vxUtils.cpp]: Keep '{' at the beginning of line to make the code look similar to other functions.

File: runvx/vxUtils.cpp (hunk 1 above)
Comment (reger-men) [reply runvx/vxUtils.cpp]: Done.

File: runvx/vxUtils.cpp (hunk 2 above)
Comment (reger-men) [reply runvx/vxUtils.cpp]: Done.

**Pull request merged**